pkgs/tools/wayland/wlprop/default.nix

```diff
@@ -0,0 +1,36 @@
+{ fetchgit, gawk, jq, lib, makeWrapper, slurp, stdenv, sway }:
+
+stdenv.mkDerivation rec {
+  pname = "wlprop";
+  version = "2022-08-18";
+
+  src = fetchgit {
+    url = "https://gist.github.com/f313386043395ff06570e02af2d9a8e0";
+    rev = "758c548bfb4be5b437c428c8062b3987f126f002";
+    sha256 = "sha256-ZJ9LYYrU2cNYikiVNTlEcI4QXcoqfl7iwk3Be+NhGG8=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ gawk jq slurp sway ];
+
+  dontBuild = true;
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    install -Dm755 wlprop.sh $out/bin/wlprop
```

Comment:
```suggestion
    install -dDm755 wlprop.sh $out/bin/wlprop
```

Reply:
Does not build for me, but was able to drop mkdir without adding "d" without any issues:
```
install: cannot create directory 'wlprop.sh': File exists
```